client/components/mobile/IconButton.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import Button from '../../common/Button';
+
+const ButtonWrapper = styled(Button)`
+width: 3rem;
+> svg {
+  width: 100%;
+  height: 100%;
+}
+`;
+
+const IconButton = props => (<ButtonWrapper
+  iconBefore={props.children}
+  kind={Button.kinds.inline}
+  {...{ ...props, children: null }}
```

Comment:
Do you need to put `children: null` here? Isn't `children` null here if you just don't put anything in between `<ButtonWrapper></ButtonWrapper>`?

Reply:
Yes, it's null if I don't pass anything, but in this case I wanted to pass `children` into the `iconBefore` prop, and not into the body of the component.
I thought of a better solution, so I'll change this soon

Reply:
I introduced an `icon` prop, now it looks a lot cleaner